.gitignore

```diff
@@ -20,6 +20,7 @@ r?
 # Build products
 tmp/*
 build?*/
+build/.vs/*
```

Comment:
Please do not push changes that don't belong to hunterization:
* https://docs.hunter.sh/en/latest/creating-new/patch.html